src/pages/RequisitionsPage.js

```diff
@@ -31,7 +31,8 @@ export class RequisitionsPage extends React.Component {
       isRequestView: true,
       isCreatingRequisition: false,
     };
-    this.requisitions = props.database.objects('Requisition');
+    this.requisitions = props.database.objects('Requisition')
+                                      .filtered('serialNumber != "-1"');
```

Comment:
Yeah I worked that out after a bit...sorry for sending you astray. I don't think you should at this stage, but if you were going to add to the UIDatabase, I'd go in the other direction...keep `Requisition` as the 'raw' type, and add a new type for the filtered requisitions.

P.S. listening carefully to this, you might hear a hint that the postSyncProcessor shouldn't be using the UIDatabase ;-)